cheri-hybrid-c-guide.tex

```diff
@@ -304,7 +305,8 @@ \subsection{Example code}
 code, and for most intents and purposes behaves identically.
 The principal difference is that annotation of pointer types with \uucap{}
 causes pointers of those types to be implemented using architectural
-capabilities, and that various \purecapCOrCpp{} APIs are available:
+capabilities, so to be subject to capability semantics including checks upon
```

Comment:
I like the change, but the grammar ("so to be") is slightly off. How about `s/so to be/which are/`?